arangodb-net-standard/IArangoDBClient.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using ArangoDBNetStandard.AqlFunctionApi;
+using ArangoDBNetStandard.AuthApi;
+using ArangoDBNetStandard.CollectionApi;
+using ArangoDBNetStandard.CursorApi;
+using ArangoDBNetStandard.DatabaseApi;
+using ArangoDBNetStandard.DocumentApi;
+using ArangoDBNetStandard.GraphApi;
+using ArangoDBNetStandard.TransactionApi;
+using ArangoDBNetStandard.UserApi;
+
+namespace ArangoDBNetStandard
+{
+    interface IArangoDBClient : IDisposable
```

Comment:
Should it be `public`? I think it defaults to internal otherwise. 🤔

Reply:
True